### ISSUE TITLE
Missing libpcre2-16-0.dll from both latest Nightly and Canary builds

### ISSUE THREAD

**Issue by Hexagon12** [citra-emu/citra#3036]
Using both Nightly-aee8bf6 and Canary-c77463f

The program instantly quits after requiring libpcre2-16-0.dll.

Related to MinGW builds, MSVC builds don't provide the error.
![citra-qt_2017-10-23_19-40-34](https://user-images.githubusercontent.com/25362478/31903365-94a75fd8-b830-11e7-942f-39b4933cf152.png)

**Comment by jroweboy** [comment]
@Hexagon12 thanks for noticing this. this error tends to happen when appveyor updates msys2 on their machine and since we don't do a static build, we have to hardcode dependencies, meaning citra may randomly break like this. The solution is to add the missing dll to the copy step. Please double check that this is the only missing dll (add it locally) and submit a pull request to add any of the missing dlls to build. 

See: https://github.com/citra-emu/citra/blob/master/appveyor.yml#L121 for the current list of dlls

**Comment by Hexagon12** [comment]
Fixed by #3037, waiting for the builds to arrive.

**Comment by Dragios** [comment]
Should be related to the missing audio as I noticed the updated MinGW dependency no longer generates this error message.
```
C:/projects/citra/src/audio_core/hle/source.cpp: In member function 'void DSP::HLE::Source::Reset()':
C:/projects/citra/src/audio_core/hle/source.cpp:45:14: warning: converting to 'std::priority_queue<DSP::HLE::Source::Buffer, std::vector<DSP::HLE::Source::Buffer>, DSP::HLE::Source::BufferOrder>' from initializer list would use explicit constructor 'std::priority_queue<_Tp, _Sequence, _Compare>::priority_queue(const _Compare&, _Sequence&&) [with _Tp = DSP::HLE::Source::Buffer; _Sequence = std::vector<DSP::HLE::Source::Buffer>; _Compare = DSP::HLE::Source::BufferOrder]'
     state = {};
              ^
```

**Comment by MerryMage** [comment]
Closed by #3037.
